src/coreComponents/linearAlgebra/interfaces/petsc/PetscPreconditioner.hpp

```diff
@@ -102,6 +111,9 @@ class PetscPreconditioner final : public PreconditionerBase< PetscInterface >
 
   /// Pointer to the PETSc implementation
   PC m_precond;
+
+  /// Pointer to the near null space
+  MatNullSpace m_nullsp = nullptr;
```

Comment:
We should stick to one style for initializing members, and I think we generally prefer in-constructor to at-declaration initializers.

Reply:
Ok.